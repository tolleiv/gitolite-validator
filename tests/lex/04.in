repo @all
    C       =   @devs
    RW+     =   @users
    config gitweb.url = git@git.example.org:%GL_REPO