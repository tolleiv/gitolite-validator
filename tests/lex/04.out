{t_string repo}
{t_group all}
{t_eol }
{t_string C}
{t_assign }
{t_group devs}
{t_eol }
{t_string RW+}
{t_assign }
{t_group users}
{t_eol }
{t_string config}
{t_string gitweb.url}
{t_assign }
{t_string git@git.example.org:%GL_REPO}
